pkg/util/deployment/deployment.go

```diff
@@ -40,6 +40,12 @@ import (
 const (
 	// The revision annotation of a deployment's replica sets which records its rollout sequence
 	RevisionAnnotation = "deployment.kubernetes.io/revision"
+	// DesiredReplicasAnnotation is the desired replicas for a deployment recorded as an annotation
+	// in its replica sets.
+	DesiredReplicasAnnotation = "deployment.kubernetes.io/desired-replicas"
```

Comment:
Anyway, if we did such a thing it still wouldn't use these annotations because they are deployment-specific. This annotation's purpose is to record what fraction of a whole **deployment** this replica set is intended to represent.